README.md

```diff
@@ -49,3 +50,19 @@ When a commit is pushed into `X.Y`:
 ## License
 
 See LICENSE files in individual subdirectories
+
+## The process (TODO)
+
+This is what we are working on right now:
+
+* [x] Initial import of Provisioning Guide
+* [x] Provide Makefiles and Travis integration
+* [x] Replace Satellite 6 with Foreman term
+* [ ] Modularize content
+* [ ] Hide irrelevant chapters
+* [ ] Incorporate parts from upstream docs
```

Comment:
https://community.theforeman.org/t/foreman-manual-reboot/22606 might be a useful link